src/k8s/pkg/k8sd/features/contour/register.go

```diff
@@ -11,5 +11,6 @@ func init() {
 		fmt.Sprintf("%s:%s", envoyImageRepo, envoyImageTag),
```

Comment:
envoyImageRepo is now just `canonical/k8s-snap/bitnami/envoy`, which is not a valid image.